### PR TITLE
adding additional port to service.yaml

### PR DESCRIPTION
**Pull request by rawanmostafahassan**
## Checklist

- [x] Meaningful PR title
- [ ] Updated README
- [ ] Github actions are passing
- [ ] I have added the Jira task

### CODE REVIEW

File: charts/service/templates/service.yaml [hunk 1]
```diff
@@ -3,13 +3,19 @@ kind: Service
...
   ports:
     - port: {{ .Values.service.port }}
       targetPort: {{ .Values.service.targetPort }}
       protocol: TCP
       name: http
+    {{- if .Values.extraService }}
+    - port: {{ .Values.extraService.Port }}
+      targetPort: {{ .Values.extraService.targetPort }}
+      protocol: TCP
+      name: metrics
```
Comment (13sh) [inline charts/service/templates/service.yaml:18]: this should also be a named property

**Review by 13sh — CHANGES_REQUESTED** [review]

File: charts/service/templates/service.yaml (hunk 1 above)
Comment (rawanmostafahassan) [reply charts/service/templates/service.yaml:18]: i named it extraService to be a general name not specified for a specific function

Comment (rawanmostafahassan) [reply charts/service/templates/service.yaml:18]: is this what you meant?

Comment (13sh) [reply charts/service/templates/service.yaml:18]:
```
 name: metrics
```

**Review by 13sh — APPROVED** [review]

**Review by Mhm0ud — CHANGES_REQUESTED** [review]
add the new variable to values, and set set the default to false, then add it to the readme file please

**Comment by rawanmostafahassan** [comment]
values added and readme updated but there is no boolean value true/false to be by default false. it will check if this value existed it will render it, if not so skip it.

**Pull request merged**